pydicer/input/filesystem.py

```diff
@@ -1,14 +1,19 @@
 from pydicer.input.base import InputBase
 
 
-class FilesystemInput(InputBase):
-    def __init__(self, working_directory=None):
+class FileSystemInput(InputBase):
+    def __init__(self, directory):
         """
         Class for inputing files from the file system
 
         Args:
-            working_directory (str|pathlib.Path, optional): The working directory in which to
-            store the data fetched. Defaults to a temp directory.
+            directory (str|pathlib.Path): The directory in which to find DICOM files.
         """
 
-        super().__init__(working_directory)
+        super().__init__(directory)
+
+        if not self.working_directory.exists():
+            raise FileNotFoundError("The directory provided does not exist")
```

Comment:
Better to start handling errors too, I guess that'll be the next step for the remaining input classes too

Reply:
Yep, we can have some discussion on how errors should best be handled. I added this in because for this input class, if a directory is specified that doesn't exist then that's not valid here and an exception is raised with a descriptive message as to what is wrong.